app/routes.js

```diff
@@ -157,10 +180,10 @@ export default (config) => {
         .sort()
         .map(mode => (mode === 'CITYBIKE' ? 'BICYCLE_RENT' : mode))
         .join(',')
-      : undefined,
+      : settings.modes,
       date: time ? moment(time * 1000).format('YYYY-MM-DD') : undefined,
       time: time ? moment(time * 1000).format('HH:mm:ss') : undefined,
-      walkReluctance: walkReluctance ? Number(walkReluctance) : undefined,
+      walkReluctance: walkReluctance ? Number(walkReluctance) : settings.walkReluctance,
       walkBoardCost: walkBoardCost ? Number(walkBoardCost) : undefined,
```

Comment:
Why setttings.walkReluctance is applied here, but walkBoardcost, walkSpeed  and minTransferTime do not default to settings?